lib/js/atoms/Card/Card.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+	<div class="card">
```

Comment:
It conflicts with https://github.com/jgthms/bulma/blob/243880f004343a41b2dc95003d2b1530900de118/sass/components/card.sass#L44. Isn't that a problem?

Reply:
Also, it's safer to use multi word component names: https://v3.vuejs.org/style-guide/#multi-word-component-names-essential
We have a lot of single word component names, but this might be the first step to a better future 🙂

Reply:
What about bulma global styles? Do we override everything?

Reply:
![Zaznaczenie_529](https://user-images.githubusercontent.com/2725894/141462574-be94e24c-4caf-4f56-b7c2-27193f0c7971.png)

Bulma styles aren't visually breaking anything right now. We can handle that in another PR. Discussion started in https://bethinkteam.slack.com/archives/CFBLF5RGA/p1636718839031100